examples/permissions/set_permissions_example.py

```diff
@@ -27,7 +27,7 @@
 Set and query permissions
 =========================
 
-Example script to set and query permissions on files and
+This example script sets and queries permissions on files and
```

Comment:
It seems odd to me that so many sections only have one example. Is this because more are to be added soon? If not, should we come up with ways of grouping them?

Reply:
@PipKat, there is currently no plan to add more examples. Do you have suggestions for grouping them any different from how they currently are?

Reply:
@saimanikant I don't feel like I have enough knowledge on the library to make a suggestion on logical example groupings. If you can't come up with any, I guess we have all these sections with only one example in each. However, it takes up a lot of space on the page, resulting in lots of unneeded scrolling.